<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="mainWidget">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QLineEdit" name="serverLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="locationLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="portLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="commentLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="serverLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Server:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="commentLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
    <widget class="QLabel" name="portLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Login:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="locationLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="savePasswordCheckBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save password ?</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="chatWidget">
    <attribute name="title">
     <string>Chat</string>
    </attribute>
    <widget class="QCheckBox" name="typingStatusCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Notify contacts when you are typing</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chatEscapeCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close chat window with escape key</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="smileysCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable smileys on chat windows</string>
     </property>
     <property name="icon">
      <iconset resource="../Images.qrc">
       <normaloff>:/images/grin.png</normaloff>:/images/grin.png</iconset>
     </property>
    </widget>
    <widget class="QComboBox" name="autoReplyComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Locked</string>
      </property>
      <property name="icon">
       <iconset resource="../Images.qrc">
        <normaloff>:/images/lock.png</normaloff>:/images/lock.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Away</string>
      </property>
      <property name="icon">
       <iconset resource="../Images.qrc">
        <normaloff>:/images/away.png</normaloff>:/images/away.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Server</string>
      </property>
      <property name="icon">
       <iconset resource="../Images.qrc">
        <normaloff>:/images/server.png</normaloff>:/images/server.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="autoReplyLineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>211</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>300</number>
     </property>
    </widget>
    <widget class="QLabel" name="autoReplyLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>341</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto reply on this state:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="advancedWidget">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
    <widget class="QLabel" name="proxyLoginLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Login:</string>
     </property>
    </widget>
    <widget class="QLabel" name="proxyPortLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="proxyPasswordLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="proxyLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="proxyLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Proxy:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="proxyPortLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>94</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="proxyLoginLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="proxyPasswordLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="proxyCheckBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>use proxy connection</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>serverLineEdit</tabstop>
  <tabstop>portLineEdit</tabstop>
  <tabstop>loginLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>locationLineEdit</tabstop>
  <tabstop>commentLineEdit</tabstop>
  <tabstop>savePasswordCheckBox</tabstop>
  <tabstop>proxyCheckBox</tabstop>
  <tabstop>proxyLineEdit</tabstop>
  <tabstop>proxyPortLineEdit</tabstop>
  <tabstop>proxyLoginLineEdit</tabstop>
  <tabstop>proxyPasswordLineEdit</tabstop>
  <tabstop>okPushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../Images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>proxyLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>proxyPortLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>proxyLoginLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>proxyPasswordLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
